interface-definitions/container.xml.in

```diff
@@ -316,6 +316,30 @@
             </properties>
             <defaultValue>on-failure</defaultValue>
           </leafNode>
+          <leafNode name="uid">
+            <properties>
+              <help>User id this container will run as</help>
+              <valueHelp>
+                <format>u32:0-65535</format>
+                <description>User id this container will run as</description>
+              </valueHelp>
+              <constraint>
+                <regex>(0|[1-9]\d{0,4}|[1-5]\d{4}|6[0-5]\d{3}|655[0-3][0-5])</regex>
```

Comment:
Is there a particular reason to use this complex regex over our range validation?

`<validator name="numeric" argument="--range 1-65535"/>`

Reply:
I'm still pretty new to the codebase and didn't notice that option. Fixed